distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+* Before this update, when a TempoStack was created on an {product-title} cluster with only an IPV6 networking stack, the compactor and ingestor pods ran in the `CrashLoopBackOff` state, resulting in multiple errors. This update provides support for IPv6 clusters.(link:https://issues.redhat.com/browse/TRACING-3226[TRACING-3226])
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_known-issues"]
+=== Known issues
+
+//There is currently a known issue:
+There are currently known issues:
+
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+// Jaeger section
+[id="distributed-tracing-rn_3-1_jaeger-release-notes"]
+== {JaegerName}
+
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_support-for-elasticsearch-operator"]
+=== Support for Elasticsearch Operator
+
+{JaegerName} 3.1 is supported for use with the OpenShift Elasticsearch Operator 5.6, 5.7, and 5.8.
+
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_deprecated-functionality"]
+=== Deprecated functionality
+
+In the {DTProductName} 3.1, Jaeger and Elasticsearch are deprecated, and both are planned to be removed in a future release. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements.
+
+In the {DTProductName} 3.1, Tempo provided by the {TempoOperator} and the OpenTelemetry Collector provided by the {OTELName} are the preferred Operators for distributed tracing collection and storage. The OpenTelemetry and Tempo distributed tracing stack is to be adopted by all users because this will be the stack that will be enhanced going forward.
+
+////
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+This update introduces the following enhancements for the {JaegerShortName}:
+* Support for ...
+////
+
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fix for the {JaegerShortName}:
+
+* Before this update, the connection target URL for the `jaeger-agent` container in the `jager-query` pod was overwritten with another namespace URL in {product-title} 4.13. This was caused by a bug in the sidecar injection code in the `jaeger-operator`, causing nondeterministic `jaeger-agent` injection. With this update, the operator prioritizes the Jaeger instance from the same namespace as the target deployment. (link:https://issues.redhat.com/browse/TRACING-3722[TRACING-3722])
```

Comment:
```suggestion
* Before this update, the connection target URL for the `jaeger-agent` container in the `jager-query` pod was overwritten with another namespace URL in {product-title} 4.13. This was caused by a bug in the sidecar injection code in the `jaeger-operator`, causing nondeterministic `jaeger-agent` injection. With this update, the Operator prioritizes the Jaeger instance from the same namespace as the target deployment. (link:https://issues.redhat.com/browse/TRACING-3722[TRACING-3722])
```

Reply:
Thank you. Done when fixing it in response to your previous comment.